srcpkgs/pwsafe/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pwsafe'
+pkgname=pwsafe
+version=0.2.0+20181220
+revision=1
+_commit=1dbcfd01a502d1579a80c208662add64b4090818
+wrksrc="${pkgname}-${_commit}"
+build_style=gnu-configure
+hostmakedepends="automake"
+makedepends="libressl-devel $(vopt_if x11 libXmu-devel)"
+short_desc="Commandline tool compatible with Counterpane's Passwordsafe"
+maintainer="gt7-void <gt7@mail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/nsd20463/pwsafe/"
+distfiles="${homepage}/releases/${pkgname}-${version}.tar.gz"
+distfiles="${homepage}/archive/${_commit}.tar.gz"
```

Comment:
setting the variable twice is bad. you can leave a comment if you expect an update to allow you to use a tagged release.